### PR TITLE
[CMR] M3-4361: Change Linode status pill to new designs

### PR DESCRIPTION
**Pull request by tiffwong**
Updated the status pill designs and removed the opacity when the settings menu is open. Still need to add a corresponding background color for dark mode.

### CODE REVIEW

File: packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.styles.ts [hunk 1]
```diff
@@ -66,6 +66,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     },
     '&:focus': {
       backgroundColor: theme.bg.primaryNavActiveBG
+    },
+    '&:first-of-type': {
+      marginLeft: 15
```
Comment (tiffwong) [inline packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.styles.ts:71]: Fixed the lack of spacing between the logo and the Dashboard button

File: packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.styles.ts [hunk 2]
```diff
@@ -117,15 +120,12 @@ const useStyles = makeStyles((theme: Theme) => ({
     padding: 8,
     position: 'absolute',
     backgroundColor: theme.bg.navy,
-    border: '1px solid #999',
     outline: 0,
     boxShadow: 'none',
+    marginTop: 3,
```
Comment (tiffwong) [inline packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.styles.ts:125]: Removed the border to make it match the rest of the designs and lined it up against the bottom of the nav

File: packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.styles.ts [hunk 3]
```diff
@@ -41,18 +41,35 @@ const styles = (theme: Theme) =>
       marginLeft: 15
     },
     chip: {
-      color: '#fff',
+      backgroundColor: '#f7f7f8',
+      color: '#5d646f',
       fontSize: '1.1rem',
-      padding: 10
+      fontFamily: theme.font.bold,
```
Comment (WilkinsKa1) [inline packages/manager/src/features/linodes/LinodesLanding/LinodesLanding.styles.ts:47]: For these base chip styles, can we create a variable in the theme file so we can spread them to other elements more easily throughout the app? I noticed the old status pill styles are still applied to the EntityDetail component. (If you search for `genericLinkStyle` throughout the app, that's a good example of how we can approach this.)

**Comment by DevDW** [comment]
Maybe make the pill text white in dark mode? It's a little hard to read in the grey color

Screen Shot 2020-07-15 at 4 12 38 PM

**Pull request merged**